pkg/controllers/disruption/metrics.go

```diff
@@ -30,6 +30,11 @@ const (
 	consolidationTypeLabel       = "consolidation_type"
 )
 
+func init() {
+	ConsolidationTimeoutsTotal.Add(0, map[string]string{consolidationTypeLabel: "multi"})
```

Comment:
nit: Do we have consts that we can use for these?

Reply:
We have these https://github.com/kubernetes-sigs/karpenter/blob/d3907afb22aecfdf9a6f62ec22cff6169ba9b60a/pkg/controllers/disruption/multinodeconsolidation.go#L226 but not consts